nmclicktests/environment.js

```diff
@@ -255,9 +256,10 @@ class Environment {
         }
       }
     });
-    ungitServer.stderr.on("data", (data) => {
-      console.log(prependLines('[server ERROR] ', data.toString()));
-      if (data.indexOf("EADDRINUSE") > -1) {
+    ungitServer.stderr.on("data", (stderr) => {
+      const stderrStr = stderr.toString();
+      console.err(prependLines('[server ERROR] ', stderrStr));
```

Comment:
Yup!, I was too haste and it is now fixed in `d130a7a`.

Thanks for catching my sloppiness and doing code review!